github/ci/prow/files/config.yaml

```
@@ -195,6 +195,9 @@ branch-protection:
       repos:
         kubevirt:
           protect: true
+          required_status_checks:
```

Maybe we can just name the last 10  release branches explicitly here and not protect master and new releases?

---

I thought we just were talking about removing the required-status-checks: for master only? So you are saying we should also remove the **protection** for master?

I understood we should do sth like this, no?
```suggestion
          branches:
            release-0.33:
              required_status_checks:
                contexts:
                - continuous-integration/travis-ci/pr
            release-0.32:
              required_status_checks:
                contexts:
                - continuous-integration/travis-ci/pr
            ...
```

---

Never mind, we can do that later.

---

No wait, yes please do it like this. I don't want to disturb tide more than necessary.